civis/utils/_jobs.py

```diff
@@ -1,7 +1,11 @@
+import logging
```

Comment:
Great, you've already moved this over to `_jobs.py`.  Will you expose this in `civis.utils.__init__` so it can be imported with `from civis.utils import run_template` ?

Reply:
Done